### PR TITLE
Update README.md with instructions to monetize video

### PR DESCRIPTION
**Pull request by jasti**

### CODE REVIEW

File: ads/README.md [hunk 1]
```diff
@@ -248,3 +249,13 @@ If you have non-trivial logic in `/ads/yournetwork.js`, adding a unit test at `/
 
 - Please consider implementing the `render-start` and `no-content-available` APIs (see [Available APIs](#available-apis)), which helps AMP to provide user a much better ad loading experience.
 - [CLA](../CONTRIBUTIONG.md#contributing-code): for anyone who has trouble to pass the automatic CLA check in a pull request, try to follow the guidelines provided by the CLA Bot. Common mistakes are 1) used a different email address in git commit; 2) didn't provide the exact company name in the PR thread. 
+
+## Serving video ads 
+AMP natively supports [a number of](https://www.ampproject.org/who/#video) video players like BrightCove, DailyMotion etc who can monetize ads.
```
Comment (cramforce) [inline ads/README.md]: This is not the right URL. Unfortunately there isn't one, I think.
It is basically: https://www.ampproject.org/docs/reference/extended.html "Search for player and a few special cases"

Comment (jasti) [inline ads/README.md]: done.

File: ads/README.md [hunk 2]
```diff
@@ -248,3 +249,13 @@ If you have non-trivial logic in `/ads/yournetwork.js`, adding a unit test at `/
 
 - Please consider implementing the `render-start` and `no-content-available` APIs (see [Available APIs](#available-apis)), which helps AMP to provide user a much better ad loading experience.
 - [CLA](../CONTRIBUTIONG.md#contributing-code): for anyone who has trouble to pass the automatic CLA check in a pull request, try to follow the guidelines provided by the CLA Bot. Common mistakes are 1) used a different email address in git commit; 2) didn't provide the exact company name in the PR thread. 
+
+## Serving video ads 
+AMP natively supports a number video players like BrightCove, DailyMotion etc that can monetize ads. For a full list, go [here] (https://www.ampproject.org/docs/reference/extended.html) and look under the 'Audio/Video' section.
```
Comment (cramforce) [inline ads/README.md]: Ah, we have a section :)

File: ads/README.md [hunk 3]
```diff
@@ -248,3 +249,13 @@ If you have non-trivial logic in `/ads/yournetwork.js`, adding a unit test at `/
...
+
+If you use a player that is not supported in AMP, you can serve your custom player using [amp-iframe](https://ampbyexample.com/components/amp-iframe/).
+
+When using amp-iframe approach:
```
Comment (cramforce) [inline ads/README.md]: Wrap `amp-iframe` in back tickss

File: ads/README.md (hunk 2 above)
Comment (lannka) [inline ads/README.md]: maybe just point to the github page instead? ../extensions/README.md#audiovideo

Comment (jasti) [inline ads/README.md]: :)

File: ads/README.md (hunk 3 above)
Comment (jasti) [inline ads/README.md]: done.

**Comment by jasti** [comment]
Closing in favor of https://github.com/ampproject/amphtml/pull/5731

**Pull request closed without merging**